### DISCUSSION TITLE
Source to Greenplum

### DISCUSSION THREAD

**Discussion by tankoand** [MaterializeInc/materialize#11660] [Q&A] [answered]
Hello. We have a Greenplum 5 (Postgres 8) DB to which we would like to connect from Materialize as a source. Unfortunately, you only support versions 10+, so I'm wondering if there is any chance you will support older versions. Also, any suggestions/hints how to somehow connect the DB and operate over its data with Materialize are deeply appreciated!

**Comment by bobbyiliev** [comment, marked as answer]
Hi there @tankoand,

The main reason for this version requirement is that PostgreSQL’s [logical decoding](https://www.postgresql.org/docs/current/static/logicaldecoding-explanation.html) feature has been introduced in version 9.4 and this is basically the mechanism that allows the extraction of the changes that were committed to the transaction log and the processing of these changes.

As a workaround you could use a CDC tool like [Debezium](https://debezium.io/releases/1.8/) or [AirByte](https://docs.airbyte.com/integrations/sources/postgres), however, I believe that they too might have a limitation for the supported Postgres version.

I believe that support for older versions is not on the roadmap anytime soon. I will bring it up internally!
